soc/nordic/common/uicr/gen_uicr.py

```diff
@@ -169,13 +149,25 @@ def main() -> None:
         "--out-uicr-hex",
         required=True,
         type=argparse.FileType("w", encoding="utf-8"),
-        help="Path to write the generated UICR HEX file to",
+        help="Path to write the generated merged UICR+PERIPHCONF HEX file to (typically zephyr.hex)",
```

Comment:
These artifacts are useful to have separate depending on your application, so the choice to keep them separate was deliberate. Can we keep them as two different outputs and use something like mergehex afterwards instead?

Reply:
customers might bundle the periphconf in their ota image but not the rest of the uicr

Reply:
I see, I did not think of that, will fix.